react/geolocation/geolocationAutoCompleteAddress.js

```diff
@@ -18,7 +18,7 @@ export default function geolocationAutoCompleteAddress(
     setCountry,
     setAddressQuery,
     (updatedAddress) =>
-      addNewField(updatedAddress, 'geolocationAutoCompleted', true),
+      addNewField(updatedAddress, 'geolocationAutoCompleted', (fieldValue) => { return (fieldValue?.value !== null) }),
```

Comment:
you can omit the `return` here by removing the curly braces, which looks a bit cleaner :-)
```suggestion
      addNewField(updatedAddress, 'geolocationAutoCompleted', (fieldValue) => fieldValue?.value !== null),
```